### ISSUE TITLE
Hc1SFMode and onetimeventilation

### ISSUE THREAD

**Issue by salvq** [guillaumezin/DomoticzEbusd#9]
I am trying to set Hc1SFMode value to `onetimeventilation` however it is always returned back to none status. When this is set directly on calormatic device UI, it stays as `onetimeventilation`...

I do expect there is some condition I have to meet in order to set the mode to 7, what values has to be set in order to turn the `onetimeventilation` ON ? Or how to grab and monitor what values has been changed ?

Thanks

```
pi@rpi3-ebus:~ $ sudo ebusctl w -c f47 Hc1SFMode 7
done

pi@rpi3-ebus:~ $ sudo ebusctl r -f Hc1SFMode
none

pi@rpi3-ebus:~ $
```

```
"Hc1SFMode": {
    "name": "Hc1SFMode",
    "passive": false,
    "write": false,
    "lastup": 1604443535,
    "zz": 21,
    "id": [181, 9, 13, 93, 0],
    "fields": {
     "0": {"name": "", "value": "none"}
    },
    "fielddefs": [
     { "name": "", "slave": true, "type": "UCH", "isbits": false, "length": 1, "values": { "0": "none", "1": "party", "2": "onedaybankholiday", "3": "onedayholiday", "4": "holiday", "5": "bankholiday", "6": "quickveto", "7": "onetimeventilation", "8": "onetimewater" }, "unit": "", "comment": "activated special function of first heating circuit"}
```

**Comment by z1mEk** [comment]
This problem has already been reported, also for the ebusd configuration (https://github.com/john30/ebusd-configuration/issues/128) - unfortunately no one can help :( The "onetimewater" does not work either. This is a problem for the author of the ebusd library, not the Domoticz plugin.

**Comment by salvq** [comment]
Right, for onetimeventilation, I see when i make a change on calormatic ui, desired temp goes to 5C however i am not able to change desire temp from ebusd.

Do you know how to change desire temp for 470f? The process may work the way, change desire temp to 5C -> change status to onetimeventilation.

**Comment by guillaumezin** [comment]
Try f47:Hc1DayTemp and f47:Hc1NightTemp and f47:Hc1HolidayRoomTemp ?

**Comment by salvq** [comment]
Will try workaround and check.

This is the thing, when you do a onetimeventilation from ui, daytemp, nighttemp are kept same ie. 21 daytemp, 18 nighttemp, just desired temp changes from 21 / 18 to 5 and this is the condition i can not similate...
